### PR TITLE
feat(esp32c61): Added flasher stub support

### PR DESCRIPTION
**Pull request by jakub-kocka**
- Added flasher stub support for ESP32-C61 for UART and USB JTAG serial

### CODE REVIEW

File: flasher_stub/ld/rom_32c61.ld [hunk 1]
```diff
@@ -0,0 +1,411 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
```
Comment (radimkarnis) [inline flasher_stub/ld/rom_32c61.ld:5]: Please remove the license. It is not in any of the other ld files.

**Review by radimkarnis — COMMENTED** [review]
Left a single comment. Other than that, LGTM! Thanks.

File: flasher_stub/ld/rom_32c61.ld (hunk 1 above)
Comment (jakub-kocka) [reply flasher_stub/ld/rom_32c61.ld]: Thank you @radimkarnis for the review!

**Review by dobairoland — APPROVED** [review]
LGTM. Thank you @jakub-kocka!

**Pull request merged**